config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,44 @@
+require 'feature_management'
+
+if FeatureManagement.rails_csp_tooling_enabled?
+  Rails.application.config.content_security_policy do |policy|
+    connect_src = ["'self'", '*.nr-data.net', '*.google-analytics.com', 'us.acas.acuant.net']
+    connect_src << %w[ws://localhost:3035 http://localhost:3035] if Rails.env.development?
```

Comment:
it's a PR race! 

if #5746 lands first, we'll want to update this here as well: https://github.com/18F/identity-idp/pull/5746/files#diff-66cc7258886071348febbf60314501e02948b7d31e616fe88121ccc338ddb449R54-R57